okonomiyaki/platforms/platform.py

```diff
@@ -269,6 +269,34 @@ def epd_platform(self):
             self._epd_platform_string
         )
 
+    @property
+    def pep425_tag(self):
+        msg = "Cannot guess platform tag for platform {0!r}"
+
+        if self.family == MAC_OS_X:
+            if self.arch.name == X86:
+                return "macosx_{0}_i386".format(self.release, )
+            elif self.arch.name == X86_64:
+                return "macosx_10_6_x86_64"
+            else:
+                raise OkonomiyakiError(msg.format(self))
+        elif self.os == LINUX:
+            if self.arch.name == X86:
+                return "linux_i686"
+            elif self.arch.name == X86_64:
+                return "linux_x86_64"
+            else:
+                raise OkonomiyakiError(msg.format(self))
+        elif self.family == WINDOWS:
+            if self.arch.name == X86:
+                return "win32"
+            elif self.arch.name == X86_64:
+                return "win_amd64"
```

Comment:
Just to make sure, I have test `bdist_wheel` on the traits tarball, and the platform is indeed `win_amd64`.

Thinking about it, though, it may be batter to somehow normalize it, and only convert to this format when requesting `pep425` tag (e.g. for wheels, when we support it).